src/station/station_info.rs

```diff
@@ -0,0 +1,946 @@
+// SPDX-License-Identifier: MIT
+
+use std::convert::TryInto;
+
+use anyhow::Context;
+use bilge::prelude::*;
```

Comment:
Please be explicit on what you imported.

As I mentioned above, please try to avoid exposing bitflag to user.